tests/Doctrine/Tests/ORM/Tools/SetupTest.php

```diff
@@ -71,6 +71,22 @@ public function testYAMLConfiguration()
         $this->assertInstanceOf('Doctrine\ORM\Mapping\Driver\YamlDriver', $config->getMetadataDriverImpl());
     }
 
+    public function testConfigureCacheNamespace()
+    {
+        // automatically set namespace
+        $config = Setup::createConfiguration(false, '/foo');
+
+        $this->assertSame('dc2_1effb2475fcfba4f9e8b8a1dbc8f3caf_', $config->getMetadataCacheImpl()->getNamespace());
+
+
+        // manually set namespace
+        $cache = new ArrayCache();
```

Comment:
It would be better if we move this to another test case, to make things more explicit